04-simulate-an-election/election.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+from random import random
+
+def elect_in_region(election_result, candidate_chances):
+    if election_result <= candidate_chances:
+        return 'a'
```

Comment:
Please extract a global constant for both candidate identifiers, and use them in the places where you now use `'a'` and `'b'`.